packages/dashboard/src/components/DashboardGridLayout/DashboardGridSectionLayout.tsx

```diff
@@ -17,30 +18,7 @@ require('style-loader!css-loader!react-grid-layout/css/styles.css');
 require('style-loader!css-loader!react-resizable/css/styles.css');
 require('style-loader!css-loader!./DashboardGridLayout.css');
```

Comment:
do we need this here?

Reply:
We dont. It is used in the base class. It is not necessary here. Good catch

---
In reply to: [216083249](https://github.com/OfficeDev/office-ui-fabric-react/pull/6285#discussion_r216083249) [](ancestors = 216083249)